helm/cert-manager/Chart.yaml

```diff
@@ -10,10 +10,14 @@ sources:
 annotations:
   application.giantswarm.io/team: bigmac
 kubeVersion: ">=1.22.0-0"
+maintainers:
+  - name: Bigmac
 dependencies:
   - name: cert-manager-giantswarm-clusterissuer
     version: 2.0.0
     alias: giantSwarmClusterIssuer
   - name: cert-manager-giantswarm-netpol
-    version: 0.1.0
+    version: 0.2.0
+  - name: cert-manager-giantswarm-ciliumnetworkpolicies
+    version: 0.2.0
```

Comment:
Do we have to bump those versions?

Reply:
I don't think if this is essential I was just trying to pass linting, will revert on next commit